EIPS/eip-6188.md

```diff
@@ -0,0 +1,44 @@
+---
+eip: 6188
+title: Nonce Cap
+description: Caps the nonce at 2^64-2
+author: Pandapip1 (@Pandapip1)
+discussions-to: https://ethereum-magicians.org/t/eip-6190-functional-selfdestruct/12232
+status: Draft
+type: Standards Track
+category: Core
+created: 2022-12-20
+requires: 2929
+---
+
+## Abstract
+
+This EIP reserves a nonce for special contracts. It also caps the nonce at `2^64-2`.
+
+## Motivation
+
+This EIP is not terribly useful on its own, as it adds additional computation without any useful side effects. However, it can be used by other EIPs.
+
+## Specification
+
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+### `CREATE` and `CREATE2`
```

Comment:
This heading seems to imply that EOA transactions can still use `2^64-1`. Is that correct?

Reply:
Nope - thanks for pointing that out!